data/items/food/banana.food

```diff
@@ -0,0 +1,13 @@
+<!-- kv3 encoding:text:version{e21c7f3c-8a33-41c5-9977-a76d3a32aa0d} format:generic:version{7412167c-06e9-4698-aff2-e63eb59037e7} -->
+{
+	data = 
+	{
+		Id = "food.banana"
+		Name = "Banana"
+		Model = "models/food/banana/banana.vmdl"
+		Description = "A pretty good banana."
+		HoldType = 4
+		FeedingValue = 10
+		WasteItem = "weapon.pistol"
```

Comment:
WTF ?!! Why a pistol for the waste item ??? loooool

Reply:
OMG I'M SO DUMB